docs/local_resource.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Run Local and/or Occasional Workflows with Local Resource
+layout: docs
+---
+A Local Resource works like any other resource in your sidebar (which are most
+often some combination of Docker Build instructions and K8s Yaml)--encompasses
+a unit of work, and execute in response to file changes, or
+[manually](https://docs.tilt.dev/manual_update_control.html). For your resource
+`MyGreatService`, when one of its file dependencies changes, the work it executes
+is to build a Docker image and deploy some k8s yaml; for a Local Resource, it's
+to execute an arbitrary command on your local filesystem.
+
+# Run commands locally on file change
+
+Tilt is designed to make it easy to build images, and to sync files/run commands
+in the cloud. For some kinds of development, however, you might want to harness
+Tilt's responsiveness to file changes, but run commands locally instead. These
+cases often fall into the general categories of:
+- *file changes or other artifacts that you want on your local machine (e.g. for
+git commit)*. For instance: when `package.json` changes, run `yarn install`
+(because you want the resulting `yarn.lock` file to exist on your local computer
+so that you can commit it to git)
+
+```python
+local_resource('yarn', cmd='yarn install', deps=['package.json'])
+```
+- *tooling that you have locally but don't want to put on your container images*.
+For instance: rather than pulling/pushing big container images with the Go compiler,
+you'd rather compile your binary locally, and then use Live Update to quickly sync
+the new binary up to a running container in your cluster
```

Comment:
it's weird to have sample code for the previous one but not this one

Reply:
urghhh. but yeah you're right